QRadar: add the polling argument to the YAML (#29235)
demisto/content: Packs/QRadar/ReleaseNotes/2_4_33.md

#### Integrations

##### IBM QRadar v3

> - Fixed an issue where the following commands were not polling the task correctly:

Reviewer: ```suggestion
- Fixed an issue where polling did not work as expected in the following commands:
```

>    -  ***qradar-reference-set-value-upsert***

Reviewer: ```suggestion
   - ***qradar-reference-set-value-upsert***
```
